dipy/data/fetcher.py

```diff
@@ -392,6 +392,17 @@ def fetcher():
     doc="Download 2 subjects from the SNAIL dataset with their bundles",
     unzip=True)
 
+fetch_ivim = _make_fetcher(
+    "fetch_ivim",
+    pjoin(dipy_home, 'ivim'),
+    'https://ndownloader.figshare.com/files/',
+    ['5305243', '5305246', '5305249'],
+    ['ivim.nii.gz', 'ivim.bval', 'ivim.bvec'],
+    ['cda596f89dc2676af7d9bf1cabccf600', 
+    'f03d89f84aa9a9397103a400e43af43a',
+    'fb633a06b02807355e49ccd85cb92565'],
+    doc="Download IVIM dataset")
```

Comment:
Maybe add to the doc "Download IVIM dataset consisting of an image, bvalue and bvector files"? Not sure how specific @arokem wants these to be tho.